packages/inline-video/index.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames'
+import WistiaPlayer from 'react-player/wistia'
+import Player from 'react-player'
+import type { InlineVideoProps } from './types'
+import s from './style.module.css'
+
+export default function InlineVideo(props: InlineVideoProps) {
+  const {
+    appearance = 'light',
+    gradientSide = 'left',
+    url,
+    description,
+    solution,
+  } = props
+
+  const playerProps = {
+    controls: true,
+    url,
+    width: '100%',
+    height: '100%',
+  }
+  return (
+    <div
+      className={classNames(s.inlineVideo, s[appearance], s[gradientSide])}
+      data-testid="wpl-inline-video"
+    >
+      <div className={classNames(s.videoContainer, solution && s[solution])}>
+        <div className={s.video}>
+          {url.includes('wistia') ? (
+            <WistiaPlayer {...playerProps} />
```

Comment:
On wistia videos, I think we want to hide controls on load until video is played. Ref: https://github.com/hashicorp/hashicorp-www-next/blob/main/pages/about/index.tsx#L198-L202

```js
config={{
  options: {
    controlsVisibleOnLoad: false,
  },
}}
```

Reply:
![CleanShot 2022-11-01 at 13 16 34@2x](https://user-images.githubusercontent.com/825855/199295566-0a7a27f4-0796-45d0-99eb-a16415a4f7de.png)

Reply:
Good call, makes sense to keep that behavior consistent :+1: 

I've addressed this here:
https://github.com/hashicorp/react-components/pull/789/commits/4decdef0a19f180150dfe468ff095216e2c9e5e3

Reply:
Good point! I've addressed that here:
https://github.com/hashicorp/react-components/pull/789/commits/4decdef0a19f180150dfe468ff095216e2c9e5e3